### ISSUE TITLE
1.3.0: `cannot find type 'LayoutDirectionBehavior' in scope`

### ISSUE THREAD

**Issue by xzoky** [tevelee/SwiftUI-Flow#12]
I've updated to 1.3.0. It works fine on my Mac running Xcode 15, but my CI environment running Xcode 14 is having a hard time with the change.

```
❌  /tmp/project/SourcePackages/checkouts/SwiftUI-Flow/Sources/Flow/Internal/HFlowLayout.swift:35:41: cannot find type 'LayoutDirectionBehavior' in scope

    public var layoutDirectionBehavior: LayoutDirectionBehavior {
                                        ^~~~~~~~~~~~~~~~~~~~~~~



❌  /tmp/project/SourcePackages/checkouts/SwiftUI-Flow/Sources/Flow/Internal/Layout.swift:206:34: cannot find type 'LayoutDirectionBehavior' in scope

    var layoutDirectionBehavior: LayoutDirectionBehavior {
                                 ^~~~~~~~~~~~~~~~~~~~~~~



❌  /tmp/project/SourcePackages/checkouts/SwiftUI-Flow/Sources/Flow/Internal/VFlowLayout.swift:35:41: cannot find type 'LayoutDirectionBehavior' in scope

    public var layoutDirectionBehavior: LayoutDirectionBehavior {
                                        ^~~~~~~~~~~~~~~~~~~~~~~

```

Let me know if I can provide more context!

**Comment by tevelee** [comment]
Thanks for reporting this issue. Indeed, I started using LayoutDirectionBehavior annotated with `@available(iOS 17.0, macOS 14.0, tvOS 17.0, watchOS 10.0, *)`. I'm trying to figure out how can that cause issues for your Xcode 14 toolchain.

**Comment by tevelee** [comment]
@xzoky I published a new release 1.3.1 to unblock you until I figure this out.

**Comment by xzoky** [comment]
1.3.1 fixed it, thank you!

My project targets iOS 15.0.
I'm using Circle CI's `xcode:14.3.1` image, which is Xcode 14.3.1 (14E300b) on macOS 13.2.1, running on an Apple Silicon box.
